src/client/sound_maker.cpp

```diff
@@ -0,0 +1,99 @@
+// Luanti
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2025 cx384
```

Comment:
I don't think this is new code, thus the copyright would technically go to "various developers", right?